data/json/items/gun/22.json

```diff
@@ -9,24 +64,134 @@
     "price": 180000,
     "bashing": 8,
     "to_hit": -2,
-    "ranged_damage": 5,
-    "material": ["superalloy", "ceramic"],
+    "material": [ "superalloy", "ceramic" ],
     "symbol": "(",
     "color": "light_gray",
     "skill": "smg",
     "ammo": "22",
     "dispersion": 200,
     "sight_dispersion": 80,
-    "aim_speed" : 2,
+    "aim_speed": 2,
     "recoil": 5,
     "durability": 9,
-    "reload": 500,
-    "loudness" : 125,
     "burst": 20,
-    "valid_mod_locations": [[ "accessories", 3 ],[ "grip", 1 ],[ "mechanism", 4 ],[ "rail", 1 ],[ "sights", 1 ],[ "stock", 1 ],[ "underbarrel", 1 ]],
-    "magazines" : [
-      [ "22", [ "360_200_mag", "360_400_mag" ] ]
-    ],
-    "flags": ["MODE_BURST", "WATERPROOF_GUN", "NEVER_JAMS"]
+    "loudness": 125,
+    "reload": 500,
+    "valid_mod_locations": [ [ "accessories", 3 ], [ "grip", 1 ], [ "mechanism", 4 ], [ "rail", 1 ], [ "sights", 1 ], [ "stock", 1 ], [ "underbarrel", 1 ] ],
+    "magazines": [ [ "22", [ "360_200_mag", "360_400_mag" ] ] ],
+    "flags": [ "MODE_BURST", "WATERPROOF_GUN", "NEVER_JAMS" ]
+  },
+  {
+    "id": "ruger_1022",
```

Comment:
On Feb 11, 2016 11:52 AM, "mugling" <notifications@github.com> wrote:
>
> One day we should support a jungle clipped version (as yet I have no idea
how that would work)

A relatively terrible way to make it work would be for the "other" magazine
to nest inside the "primary" magazine, and reloading when the magazine is
in a gun would swap the contents of the two. Otherwise reloading would push
rounds into the primary, unless it's full, in which case it would push
rounds into the secondary.